graphql_compiler/tests/test_schema_fingerprint.py

```diff
@@ -0,0 +1,398 @@
+# Copyright 2020-present Kensho Technologies, LLC.
+import unittest
+
+from graphql import build_ast_schema, parse, print_schema
+
+from ..schema import compute_schema_fingerprint
+from .test_helpers import compare_graphql
+
+
+def _compare_schema_fingerprints(
+    test_case: unittest.TestCase, schema_text1: str, schema_text2: str, expect_equality: bool = True
+) -> None:
```

Comment:
I can't say I'm a fan of the name of this function or of `expect_equality = True` as an approach. A lot of the tests end in `_compare_schema_fingerprints(self, schema1, schema2)` which does not look like an assertion, and even if you assume it is one, it's not clear whether it's comparing for equality or inequality.

I'd suggest either having different functions for equality and inequality, e.g. `assert_equal_fingerprints()` and `assert_not_equal_fingerprints()`. If you are worried about the repetition between them, you could also pull `compute_schema_fingerprint(build_ast_schema(parse(x)))` into a function of its own.